tests/Integration/Behaviour/behat.yml

```diff
@@ -73,8 +74,12 @@ default:
                 - Tests\Integration\Behaviour\Features\Context\Domain\CustomerFeatureContext
                 - Tests\Integration\Behaviour\Features\Context\Domain\AddressFeatureContext
                 - Tests\Integration\Behaviour\Features\Context\Domain\OrderRefundFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\TaxFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\ProductFeatureContext
+                    - Tests\Integration\Behaviour\Features\Context\Domain\TaxFeatureContext
```

Comment:
Careful with the indentation, although if the tests are green this can be fixed later

Reply:
done ;)